oxd-server/pom.xml

```diff
@@ -21,176 +22,380 @@
         <version>4.0-SNAPSHOT</version>
     </parent>
 
-    <build>
-        <finalName>oxd-server</finalName>
+    <profiles>
+        <profile>
+            <id>built</id>
```

Comment:
we don't need copy/paste default configuration. Defaults leaves as default and it is already activated. Introduce only `windows-installer` profile with activator lets say `-Dbuild-windows-installer=true`